src/pybind/mgr/dashboard/controllers/pool.py

```diff
@@ -49,6 +49,10 @@ def list(self, attrs=None, stats=False):
         else:
             pools = CephService.get_pool_list()
 
+        for pool in pools:
+            pool['type_name'] = {1: 'replicated', 3: 'erasure'}[pool['type']]
```

Comment:
I would prefer, if we directly convert `type`, if you prefer the string in the UI:
```python
pool['type'] = {1: 'replicated', 3: 'erasure'}[pool['type']]
```